azurelinuxagent/ga/update.py

```diff
@@ -138,8 +139,13 @@ def get_update_handler():
     return UpdateHandler()
 
 
+def get_agent_global_update_signal_file():
+    return os.path.join(conf.get_lib_dir(), AGENT_UPDATE_SIGNAL_FILE)
+
+
 class UpdateHandler(object):
     TELEMETRY_HEARTBEAT_PERIOD = timedelta(minutes=30)
+    CLEAN_UPDATE_SIGNAL_PERIOD = timedelta(minutes=15)
```

Comment:
We've had the logic in place in the daemon code where we mark a version as successful only if it's been running correctly for over 15 mins. Keeping the same time check here to mark an update successful